.github/workflows-source/schedule-daily.yml

```diff
@@ -53,13 +57,30 @@ jobs:
     environment: Nightly Tests
     steps:
       - <<: *checkout
+      - <<: *generate-bazel-invocation-ids
+        with:
+          targets: |
+            [{
+              "name": "HSM Test",
+              "output_id": "hsm_test_id",
+            }, {
+              "name": "Benchmark",
+              "output_id": "benchmark_id",
+            }, {
+              "name": "HostOS metrics",
+              "output_id": "hostos_metrics_id",
+            }]
+
       - name: Run Bazel Launch Bare Metal
         shell: bash
         run: |
           echo "$ZH2_DLL01_CSV_SECRETS" > file1
           echo "$ZH2_FILE_SHARE_KEY" > file2 && chmod 400 file2
 
           launch_bare_metal() {
+            local invocation_id="$1"
+            shift
+
```

Comment:
It seems you still need to set `--invocation_id="$invocation_id"` in the bazel invocation below:

Reply:
Yup totally forgot to do it...